gaeguli/pipeline.c

```diff
@@ -447,6 +449,13 @@ _build_vsrc_pipeline (GaeguliPipeline * self, GError ** error)
         G_CALLBACK (_decodebin_pad_added), self->overlay);
   }
 
+  /* Verify whether hardware accelearted vaapijpeg decoder is available.
+   * If available, make sure that the decodebin picks it up. */
+  feature = gst_registry_find_feature (gst_registry_get (), "vaapijpegdec",
+      GST_TYPE_ELEMENT_FACTORY);
+  if (feature)
+    gst_plugin_feature_set_rank (feature, GST_RANK_PRIMARY + 100);
```

Comment:
I think it's enough for now if we are sure `vaapijpegdec` doesn't interfere with creating the existing pipeline.  What do you think? @xhaakon

Reply:
This revision LGTM